samples/openthread/cli/CMakeLists.txt

```diff
@@ -13,5 +13,5 @@ project(openthread_cli)
 target_sources(app PRIVATE src/main.c)
 # NORDIC SDK APP END
 
-target_sources_ifdef(CONFIG_BT app PRIVATE src/ble.c)
+target_sources_ifdef(CONFIG_CLI_SAMPLE_MULTIPROTOCOL app PRIVATE src/ble.c)
```

Comment:
Isn't the Kconfig update with `CLI_SAMPLE_MULTIPROTOCOL` missing?

Reply:
https://github.com/nrfconnect/sdk-nrf/pull/13391/files#diff-fe1b703e068591e6255c6d243be32d704aabe4f2fd9f280182ad867b639401f7R18